rust-runtime/aws-smithy-runtime/Cargo.toml

```diff
@@ -31,7 +31,7 @@ bytes = "1"
 # We probably need to update unit tests using the `fastrand` crate when that happens
 fastrand = "2.0.0"
 h2 = { version = "0.3", default-features = false, optional = true }
-http = { version = "0.2.8" }
+http0 = { package = "http", version = "0.2.8" }
```

Comment:
**consistency**: In `aws-runtime/Cargo.toml` this is `http-02x` here it's just `http0`, we should be consistent across the codebase.